File: CS/MyWinExplorerView/MyTileViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid.Views.WinExplorer.ViewInfo;
using DevExpress.XtraGrid.Views.WinExplorer;
using DevExpress.XtraGrid.Views.Tile.ViewInfo;
using DevExpress.XtraGrid.Views.Tile;
using DevExpress.XtraEditors;
using System.Drawing;
using DevExpress.Utils;

namespace MyXtraGrid
{
    public class MyTileViewInfo : TileViewInfo
    {
        public MyTileViewInfo(TileView view)
            : base(view)
        {
            
        }
    }
}

File: VB/MyWinExplorerView/MyTileViewInfo.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid.Views.WinExplorer.ViewInfo
Imports DevExpress.XtraGrid.Views.WinExplorer
Imports DevExpress.XtraGrid.Views.Tile.ViewInfo
Imports DevExpress.XtraGrid.Views.Tile
Imports DevExpress.XtraEditors
Imports System.Drawing
Imports DevExpress.Utils

Namespace MyXtraGrid
    Public Class MyTileViewInfo
        Inherits TileViewInfo

        Public Sub New(ByVal view As TileView)
            MyBase.New(view)

        End Sub
    End Class
End Namespace
